tests/unit/test_charm.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 dima.tisnek@canonical.com
+# See LICENSE file for licensing details.
+#
+# Learn more about testing at: https://juju.is/docs/sdk/testing
+import pytest
+from ops.testing import Harness
+from ops.model import ActiveStatus, MaintenanceStatus
```

Comment:
Our preference is to have everyone use the names from the top-level `ops` namespace (other than what's in pebble and testing). So either:

```suggestion
from ops import ActiveStatus, MaintenanceStatus
```

or just the plain `import ops` and then `ops.ActiveStatus` and `ops.MaintenanceStatus`.

Reply:
Agreed, and we've basically settled on the `import ops ... ops.ActiveStatus` style, as it's easier to see where the names came from.